src/index.js

```diff
@@ -38,6 +38,8 @@ const AtlasSDK = {
   configure(options = {}) {
     const env = options.is_sandbox ? 'staging' : 'production';
     const fileName = `${options.client_id}-${env}`;
+    const cachedConfName = `${options.client_id}-${options.sales_channel}-${env}`;
```

Comment:
Could you explain hows does this help fixing the issue?

Reply:
Already explained: https://github.com/zalando-incubator/atlas-js-core/issues/149